locale/hu.json

```diff
@@ -4,19 +4,19 @@
         "humanizer_language": "hu"
     },
     "restarter": {
-        "start_timeout": "sikertelen indítás a megadott idő intervallumon belül",
-        "crash_detected": "összeomlás észlelve",
-        "hang_detected": "lefagyás észlelve",
+        "start_timeout": "Sikertelen indítás a megadott idő intervallumon belül",
```

Comment:
The original locale has the following entries lowercased `start_timeout, crash_detected, hang_detected`, I think we should stick with that.